docker-compose.yaml

```
@@ -0,0 +1,14 @@
+services:
+  private-gpt:
+    build:
+      dockerfile: Dockerfile.local
+    volumes:
+      - ./local_data/:/home/worker/app/local_data
+      - ./models/:/home/worker/app/models
+    ports:
+      - 127.0.0.1:8080:8001
+    environment:
+      PORT: 8001
```

Here you actually want to use the usual 8080

```suggestion
      PORT: 8080
```

---

done